templates/3dsmax/scripts/RepathRenderElements.ms

```diff
@@ -21,28 +21,24 @@ if(rem != undefined) then
 
 r = renderers.current     
 rendererName = r as string
-indexVrayAdv = findString rendererName "V_Ray_Adv_"
-indexVrayNext = findString rendererName "V_Ray_Next_"
+indexVRay5 = findString rendererName "V_Ray_5_"
 
-if (indexVrayAdv == 1 or indexVrayNext == 1) then
+if (indexVRay5 == 1) then
 (
+    tokens = filterString irAutoSavePath "\\"
+    filename = tokens[tokens.count]
+    prefix = sysEnv.GetEnvironmentVariable "AZ_BATCH_TASK_WORKING_DIR"
+    newPath = prefix + "\\images\\" + filename
+
     irAutoSavePath = r.adv_irradmap_autoSaveFileName
     if(irAutoSavePath != undefined) then
```

Comment:
Watch out for the case where `irAutoSavePath` is defined as a string but empty

so in the next line `tokens` will be undefined
```
tokens = filterString irAutoSavePath "\\"
```

A fix would be
```
if(irAutoSavePath != undefined and irAutoSavePath != "") then
```